Header header
float64 distance
int64 diff_from_init

# For debug
float64 prop_const
uint32 init
